sec_certs/sample/common_criteria.py

```diff
@@ -443,6 +449,8 @@ def download_pdf_target(cert: 'CommonCriteriaCert') -> 'CommonCriteriaCert':
             error_msg = f'failed to download ST from {cert.report_link}, code: {exit_code}'
             logger.error(f'Cert dgst: {cert.dgst}' + error_msg)
             cert.state.st_download_ok = False
+            if not cert.state.errors:
+                cert.state.errors = []
             cert.state.errors.append(error_msg)
```

Comment:
These three lines could likely be simplified right? Also not sure the initial check works correctly. If the `errors` attribute is missing it will raise an exception right? Are we sure the attr is there and just not sure it is a list? What is it then?

Reply:
The problem is that `errors` is optional, therefore it can be `None`. Here we have to make sure it is a list, so we can append the error message. I think the `errors` attribute will be always present.